src/routes.jsx

```diff
@@ -0,0 +1,36 @@
+
+// LIBRARIES
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+// Components
+import Navbar from './components/Navbar/Navbar';
+// import Calculator from './components/Calculator';
+import NotFound from './components/NotFound/NotFound';
+import Homepage from './pages/Homepage';
+import Footer from './components/Footer/Footer';
+
+// Pages
+// import About from './pages/About';
+// import NotMatch from './pages/NotMatch';
```

Comment:
- [ ] It will be nice to remove commented code from your code base, as this makes your code base dirty and less readable.